### DISCUSSION TITLE
Snare Customisation

### DISCUSSION THREAD

**Discussion by OliHebden** [telekom-security/tpotce#765] [Q&A] [answered]
Hello, just recently setup tpot and I'm wanting to change the website used by snare/tanner to a website I created (consisting of an index.html, css folder and an images folder). 

I read through the snare documentation and I haven't found anything explaining how I go about this?

**Comment by t3chn0m4g3** [comment, marked as answer]
Please have a look at our [Wiki ](https://github.com/telekom-security/tpotce/wiki/Reconfigure-ews.cfg) with regard to docker volumes and review the snare/tanner [documentation ](https://snare.readthedocs.io/en/latest/) on how to clone your own sites.
